### DISCUSSION TITLE
Add display option for SSH entries?

### DISCUSSION THREAD

**Discussion by techrivertree** [keepassxreboot/keepassxc#11519] [Ideas]
I use SSH to access my headless servers so how about adding a new column option for SSH? The entry would be something like username@ipaddress (test@10.1.1125). That way we could list the ssh command in that area instead of the URL area.

**Comment by droidmonkey** [comment]
Set your url to: `cmd://ssh {USERNAME}@server`

**Comment by techrivertree** [comment]
I should of mentioned that I use Linux.

That command does not work.

![Screenshot from 2024-12-01 18-19-10](https://github.com/user-attachments/assets/f6b50a32-b14c-4cee-afca-ad514ee63bff)

**Reply by droidmonkey** [reply]
I don't know where you are trying to do that, but your image snippet doesn't look like keepassxc

**Reply by michaelk83** [reply]
The `cmd://` part is interpreted by KeePassXC when you try to open that URI from KeePassXC. You're not supposed to type the `cmd://` in bash. The bash command is just the part after that: `ssh {USERNAME}@server`.
